main/src/main/scala/org/clulab/numeric/mentions/DateRangeMention.scala

```diff
@@ -15,10 +15,11 @@ class DateRangeMention ( labels: Seq[String],
                          var date2Norm: String )
   extends TextBoundMention(labels, tokenInterval, sentence, document, keep, foundBy, attachments) with Norm {
 
-  // Change date1Norm if the date1Norm does not have a year value but date2Norm has a year value
+  // Change date1Norm if the date1Norm is not undefined (i.e., XXXX-XX-XX) and
+  // does not have a year value but date2Norm has a year value
   // This will handle date ranges of form month date and month date of year.
   // Ex: "Sowing dates between May 3 and October 2, 2020" will have a normalized value of 2020-05-03 -- 2020-10-02
-  if (date1Norm contains "XXXX") {
+  if (!date1Norm.equals("XXXX-XX-XX") && date1Norm.contains("XXXX")) {
     if (!(date2Norm contains "XXXX")) {
```

Comment:
I see that the PR was already closed, but nevertheless, could this `contains` be changed to `startsWith` for efficiency?  It would be cool if there was a class DateNorm that takes the string in the constructor and then answers questions like hasYear, isUndefined, etc., but I'll take efficiency.

Reply:
The `contains` was originally there, so I just kept it as it was, but I can change it.